spec/features/user_can_delete_only_their_posts_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+require 'pry'
+
+RSpec.feature "Delete & edit only my posts", type: :feature do
+  scenario "user only sees edit & delete links for their posts" do
+    # User 1 - sign in, create a post, sign out
+    sign_up("user1@gmail.com", "123456")
+    click_link "View Your Posts"
+    create_a_post("I love monsieur Travis")
+    click_link "Sign out"
+    # User 2 - sign in, create a post, view all posts
+    sign_up("user2@gmail.com", "123456")
+    click_link "View Your Posts"
+    create_a_post("J'adore la baguette")
+    click_link "View All Posts"
+    # grab posts from database
+    post1 = Post.all.where(message: "I love monsieur Travis")[0]
+    post2 = Post.all.where(message: "J'adore la baguette")[0]    
+    
```

Comment:
this looks pretty cool - would love an explanation in our app run through catch up